src/plugins/dashboard/server/content_management/v3/cm_services.ts

```diff
@@ -284,10 +285,11 @@ export const panelSchema = schema.object({
   type: schema.string({ meta: { description: 'The embeddable type' } }),
   panelRefName: schema.maybe(schema.string()),
   gridData: gridDataSchema,
-  panelIndex: schema.string({
-    meta: { description: 'The unique ID of the panel.' },
-    defaultValue: schema.siblingRef('gridData.i'),
-  }),
+  panelIndex: schema.maybe(
+    schema.string({
+      meta: { description: 'The unique ID of the panel.' },
```

Comment:
Should this be index instead of ID?

Reply:
If not specified or created using the UI the `panelIndex` is a randomly generated UUID. IMO, "ID" is more applicable even though the property name is `panelIndex`. 🤷